misk-feature-testing/src/test/java/misk/feature/testing/FakeFeatureFlagsJavaTest.java

```diff
@@ -64,4 +64,41 @@ public void configure() {
     assertThat(subject.getDouble(FEATURE, "joker5", attributes)).isEqualTo(1.0);
   }
 
+  @Test
```

Comment:
Can you put tests for all the types?

Reply:
Absolutely! I've just added a bunch of tests for `wisp-feature-flag` which cover all of the strong types, but I'll add the remaining types to the Java tests as well.

Reply:
This is also done